OpenRVS/classes/OpenRVS.uc

```diff
@@ -0,0 +1,33 @@
+class OpenRVS extends Actor;
```

Comment:
I really like this new class, and I hope we can extend Object instead of Actor, so we can easily utilize it for example like this:
`local OpenRVS OpenRVS;`
`OpenRVS = new class'OpenRVS';`
`something = OpenRVS.GetSomething();`
`OpenRVS.DoSomething();`
`OpenRVS = none;`
Which is simpler than needing a ViewportOwner reference and spawning an Actor class in the Entry level.

I've wanted for a while now to have a general utility class that eliminates the need for OpenFOV.  OpenFOV currently is used to read a couple vars from the config file (fov and cheat manager class), and class'OpenRVS' as a utility class could handle both of those as well as version checking, initial log file write, etc.  It makes more sense than having a single class for version checking, and a single class named "OpenFOV" for retrieving two variables.  What do you think?

Reply:
I'll take a shot at extending Object instead; I had some difficulties initially but will revisit.

The OpenFOV change sounds like a good followup, maybe best performed in a separate PR.

Reply:
Ah, I remember now:

OpenRVS calls Spawn(OpenHTTPClient). In order to call Spawn I made it an Actor. OpenHTTPClient extends TcpLink, so it _must_ be spawned as an Actor.

I can leave OpenRVS as an Actor, or I can make OpenRVS an Object and pass in the HTTP client from upstream.

Reply:
Actually, the upstream (OpenCustomMissionWidget) also extends Object and cannot call Spawn() either.

Reply:
You can call the spawn function on any actor that is passed to an object class as well though. For example, assuming openrvs extends object, add an argument for a playercontroller or levelinfo or some other actor in the Init() function. Eg
function init(Actor SomeActor)
{
   http = SomeActor.spawn(class'OpenHTTPClient';
}
Hope that makes sense, on phone right now.

Reply:
I get it now, thanks! Updated the PR to extend Object.